src/js/view/month/weekdayInMonthSchedule.hbs

```diff
@@ -30,7 +30,7 @@
                     {{#if model.isFocused}}
                         background: #ffffff
                     {{else}}
-                        background:{{model.color}}
+                        background:{{model.borderColor}}
```

Comment:
Why do you use `borderColor`? In `more.hbs` you used `bgColor` as calendar color.

Reply:
@dongsik-yoo I changed `more.hbs` to use `borderColor`.
I think bullet color should be same to border color of all-day schedule